src/support/z_pivot_rsc.erl

```diff
@@ -141,11 +141,12 @@ queue_all(Context) ->
         end.
 
 %% @doc Insert a rsc_id in the pivot queue
+-spec insert_queue(m_rsc:resource(), #context{}) -> ok | {error, eexist}.
 insert_queue(Id, Context) ->
     insert_queue(Id, calendar:universal_time(), Context).
 
 %% @doc Insert a rsc_id in the pivot queue for a certain date
--spec insert_queue(integer(), calendar:date(), #context{}) -> ok | {error, eexist}.
+-spec insert_queue(integer(), {calendar:date(), calendar:time()}, #context{}) -> ok | {error, eexist}.
```

Comment:
Line 149 is too long: -spec insert_queue(integer(), {calendar:date(), calendar:time()}, #context{}) -> ok | {error, eexist}..